src/prefect/client/orion.py

```diff
@@ -364,12 +364,17 @@ async def create_flow_run_from_deployment(
         Create a flow run for a deployment.
 
         Args:
-            deployment: The deployment model to create the flow run from
+            deployment_id: The deployment ID to create the flow run from
             parameters: Parameter overrides for this flow run. Merged with the
                 deployment defaults
             context: Optional run context data
             state: The initial state for the run. If not provided, defaults to
                 `Scheduled` for now. Should always be a `Scheduled` type.
+            name: Custom flow run name
+            tags: List of extra tags to apply to the flow run in
+                addition to `deployment.tags`
+            idempotency_key: Optional key for flow run. Used to ensure the same flow run
+                is not created multiple times.
```

Comment:
```suggestion
            name: An optional name for the flow run. If not provided, the server will
                generate a name.
            tags: An optional iterable of tags to apply to the flow run; these tags
                are merged with the deployment's tags.
            idempotency_key: Optional idempotency key for creation of the flow run.
                If the key matches the key of an existing flow run, the existing run will
                be returned instead of creating a new one.
```